### DISCUSSION TITLE
Start with mini map view(Tab key)

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#276] [Q&A] [answered]
Hi. thank you for your great SoloLeveling.
Is there way to auto show mini map when game start?

**Comment by Amis-boy** [comment]
沒有顯示

**Comment by isid0re** [comment, marked as answer]
Add the following to the sololeveling class config

`Config.AutoMap = true;`

**Reply by mirakhany** [reply]
Thank you so much!
It works!
